### PR TITLE
Fixes #896 embedded image alignment, and Fixes #900 add visual indicator on unpublished content.

### LINKED ISSUE #896: Ability to align media and text
I could not find an existing ticket for image/video alignment, so adding one. (#586 relates slightly in that we'd like the size/alignment selection to happen when uploading media, but right now there is no alignment option)

Currently inline images in the ckeditor cannot be aligned with text. This was a feature available in QS1 and is expected by content editors.

## Conditions of satisfaction
- [ ] Content editors can choose to align an image with text
- [ ] Image properly aligns with text based on selection and has appropriate margins

### LINKED ISSUE #900: Indicate visually when content is unpublished
## Motivation
Currently in QS 2 there is no way for an editor to visually see that piece of content is unpublished. This is likely to cause confusion.

### Is your feature request related to a problem? Please describe.
Content editors need to know if content is unpublished if they are viewing it from the user side.

### Describe the solution you'd like
In QS1, unpublished content has a pink background and is marked unpublished

### Roles and Permissions considerations
These roles would need to see the visual indicator
- Content editor
- Content administrator
- Administrator

### PR DESCRIPTION
**Pull request by trackleft**
## Description
Fixed unpublished glitch in CSS.
Found that style.css is being added to page.
Added margins for embedded images aligned right, left, and center.
Added ability to add alignment to the az_standard text format.

## Related Issue
#896
#900

## How Has This Been Tested?
Locally with demo content.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: themes/custom/az_barrio/css/style.css [hunk 1]
```diff
@@ -656,10 +681,8 @@ fieldset.collapsed {
     clear: initial;
   }
 }
-.field--type-image img,
-.field--name-field-user-picture img {
-  margin: initial;
-}
+
+
```
Comment (trackleft) [inline themes/custom/az_barrio/css/style.css:685]:
```suggestion
```

File: themes/custom/az_barrio/css/style.css [hunk 2]
```diff
@@ -656,10 +690,8 @@ fieldset.collapsed {
...
```
Comment (trackleft) [inline themes/custom/az_barrio/css/style.css]:
```suggestion
```

File: themes/custom/az_barrio/css/style.css [hunk 3]
```diff
@@ -641,7 +645,37 @@ fieldset.collapsed {
...
+.align-center figure {
+  margin-left: auto;
+  margin-right: auto;
+}
+.field--type-image img,
+.field--name-field-user-picture img {
+  margin: initial;
+}
 
+@media screen (min-width: 576px) {
```
Comment (trackleft) [inline themes/custom/az_barrio/css/style.css:664]:
```suggestion
@media screen and (min-width: 576px) {
```

**Comment by smayden** [comment]
images seem to be working well in the limited testing I've done.

But videos are not appear when left/right aligned. They appear in the editor but not on the saved page.

**Comment by trackleft** [comment]
This one is triggering PHPUNit errors.

**Comment by trackleft** [comment]
This is ready for review https://app.probo.ci/#/dashboard/build/c1b7071c-8ff4-4ca7-9ca5-b2fb80421e37

**Comment by camikazegreen** [comment]
The Unpublished indicator functionality looks good.

**Comment by smayden** [comment]
The changes on the video side of this PR are causing issues with the responsiveness of videos. Large/medium sized videos/images go off screen at smaller display sizes (images seem ok).

I'm also now seeing videos visually/functionally break the editing experience (can't edit the paragraph anymore):
image

https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-912.probo.build/node/15/edit?destination=/admin/content

**Comment by trackleft** [comment]
https://c1b7071c-8ff4-4ca7-9ca5-b2fb80421e37.probo.build/

**Comment by mmunro-ltrr** [comment]
The code suggestions above (now included) show that this fixes a major CSS syntax error in themes/custom/az_barrio/css/style.css (an unterminated opening `{` bracket).  The PR #869 (fixing the active state of the sidebar nav) introduced this by moving the media query `@media screen and (min-width: 60em)` away from the start of the Side Menus section in this file without moving the closing `}` bracket. But is the current position of the closing bracket correct? That's now extending the scope of the media query well beyond where it used to be before the sidebar nav fix.

File: themes/custom/az_barrio/css/style.css [hunk 4]
```diff
@@ -641,7 +645,37 @@ fieldset.collapsed {
   width: initial;
   max-width: 100%;
 }
+}
```
Comment (mmunro-ltrr) [inline themes/custom/az_barrio/css/style.css:648]: Should the media query closing bracket be this far down?

**Comment by mmunro-ltrr** [comment]
Of course Arizona Bootstrap can't be affecting anything in the editing form — that's all Claro. I turned off CSS & JS aggregation on the Probo build in any case, to see where the various files really were.

File: themes/custom/az_barrio/css/style.css (hunk 4 above)
Comment (trackleft) [reply themes/custom/az_barrio/css/style.css:648]: honestly, I don't really know where it needs to be, there wasn't one and so everything above it was already being set on that media query, and I figured  adding it there would do the least amount of harm

**Review by mmunro-ltrr — APPROVED** [review]
Seems to work now: it would be good to merge soon, & treat anything unrelated about the AZ Barrio CSS as a follow-up.

**Review by smayden — APPROVED** [review]
Tested sizing/alignment for both videos and images. Working well! Unpublished feature also working. Thanks!!

**Review by mmunro-ltrr — DISMISSED** [review]

**Review by smayden — APPROVED** [review]

File: themes/custom/az_barrio/az_barrio.libraries.yml [hunk 5]
```diff
@@ -27,4 +27,4 @@ arizona-bootstrap:
 material-design-icons-sharp:
   css:
     theme:
-      https://fonts.googleapis.com/css?family=Material+Icons+Sharp: {}
+      https://fonts.googleapis.com/css?family=Material+Icons+Sharp: {}
```
Comment (trackleft) [inline themes/custom/az_barrio/az_barrio.libraries.yml:30]:
```suggestion
      https://fonts.googleapis.com/css?family=Material+Icons+Sharp: {}
      
```

File: modules/custom/az_media/az_media.libraries.yml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+az_media_display:
+  css:
+    component:
+      css/az-media-display.css: {}
+az_media_edit:
+  css:
+    component:
+      css/az-media-edit.css: {}
```
Comment (joeparsons) [inline modules/custom/az_media/az_media.libraries.yml:8]:
```suggestion
      css/az-media-edit.css: {}

```

File: modules/custom/az_media/css/az-media-edit.css [hunk 7]
```diff
@@ -0,0 +1,8 @@
+.align-center .az-media {
+  display: block;
+  margin-right: auto;
+  margin-left: auto;
+}
+.az-media.embed-responsive {
+  width:400px;
+}
```
Comment (joeparsons) [inline modules/custom/az_media/css/az-media-edit.css:8]:
```suggestion
}

```

File: themes/custom/az_barrio/az_barrio.info.yml [hunk 8]
```diff
@@ -41,3 +41,7 @@ regions:
 ckeditor_stylesheets:
   - https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css
   - https://cdn.digital.arizona.edu/lib/arizona-bootstrap/latest/css/arizona-bootstrap.min.css
+
+libraries-extend:
+  system/base:
+    - az_media/az_media_display
```
Comment (joeparsons) [inline themes/custom/az_barrio/az_barrio.info.yml:47]:
```suggestion
    - az_media/az_media_display

```

**Comment by trackleft** [comment]
> The code suggestions above (now included) show that this fixes a major CSS syntax error in themes/custom/az_barrio/css/style.css (an unterminated opening `{` bracket). The PR #869 (fixing the active state of the sidebar nav) introduced this by moving the media query `@media screen and (min-width: 60em)` away from the start of the Side Menus section in this file without moving the closing `}` bracket. But is the current position of the closing bracket correct? That's now extending the scope of the media query well beyond where it used to be before the sidebar nav fix.

I'm not seeing the closing bracket in this issue either, but maybe I need more coffee, https://github.com/az-digital/az_quickstart/pull/869/files

**Review by joeparsons — APPROVED** [review]

**Review by mmunro-ltrr — APPROVED** [review]

**Review by smayden — APPROVED** [review]

**Pull request merged**